packages/skeleton-svelte/src/lib/components/Tab/TabsPanel.svelte

```diff
@@ -0,0 +1,21 @@
+<script lang="ts">
+	import type { TabsPanelProps } from './types.js';
+
+	let {
+		value,
+		group,
+		// Root
+		base = '',
```

Comment:
If there are no inherent base/classes styles then having both feels redundant. I'd say we default to one or the other. `bases` seems most obvious.

Reply:
I would actually suggest to keep classes instead, since users will most likely avoid writing to base because in most of the other components we have some styles in base, so we will get a lot of questions on where is classes for this component.
Again let me know if you are not OK with the decision and I will change it.